pepper-apis/docs/specification/src/endpoints/studies.invitations.yml

```diff
@@ -0,0 +1,109 @@
+parameters:
+  - in: path
+    name: study
+    required: true
+    description: the study guid
+    schema:
+      type: string
+  - in: path
+    name: invite
+    required: true
+    description: the invitation id
+    schema:
+      type: string
+
+get:
+  operationId: getInvitation
+  tags:
+    - Invitations
+  summary: GetInvitation
+  description: |
+    Fetch an invitation, along with associated user data.
+
+    Note: this API is restricted to study admins.
+  responses:
+    200:
+      description: invitation details
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - invitationId
+              - createdAt
+              - voidedAt
+              - verifiedAt
+              - acceptedAt
+              - notes
+              - userGuid
+              - userHruid
+              - userLoginEmail
+            properties:
+              invitationId:
+                description: the id of the invitation
+                type: string
+              createdAt:
+                description: UTC timestamp of when invitation was created, in ISO-8601 format
+                type: string
+                example: 2020-05-13T17:58:33.887Z
+              voidedAt:
+                description: UTC timestamp of when invitation is voided
+                type: string
+                nullable: true
+              verifiedAt:
```

Comment:
What is the type of this? A timestamp in UTC should be an integer, not a string, unless we're using an ISO 8601 datetime here in the UTC zone

Reply:
fixed